src/pages/SignUp/SignUp.js

```diff
@@ -1,7 +1,148 @@
 import React, { Component } from 'react';
+import { chkEmail, chkPwd, chkName, chkAddress } from './Validation';
+import '../SignUp/SignUp.scss';
 
 export default class Login extends Component {
+  emailRef = React.createRef();
+  pwRef = React.createRef();
+
+  state = {
+    name: '',
+    email: '',
+    pw: '',
+    address: '',
+  };
+
+  emailInput = email => {
+    this.setState({
+      email: email.target.value,
+    });
+  };
```

Comment:
```suggestion
  emailInput = event => {
    const { name, value } = event.target
    
    this.setState({ [name]: value });
  };
```
객체 계산된 속성명 문법을 사용하면 input 태그들을 핸들링하는 함수를 하나로 합칠 수 있을 것 같습니다

Reply:
> 객체 계산된 속성명 문법을 사용하면 input 태그들을 핸들링하는 함수를 하나로 합칠 수 있을 것 같습니다

4개의 Input 함수 하나로 통합 하여 수정 후 push 하였습니다.